templates/section_pages/support_pages.hbs

```diff
@@ -0,0 +1,203 @@
+<div class="container-divider"></div>
+<div class="container"> 
+    <nav class="sub-nav custom-page-search-mini">
+        {{breadcrumbs}}
+        {{search scoped=true submit=false placeholder='Search for help articles, videos and more'}}
+    </nav>
+    <header class="page-header">
+        <span>
+            <h1>{{section.name}}</h1>
+            <p class="custom-contact-header-additional-desc">
+                <b>{{section.description}}</b>
+            </p>
+            <h2>Contact us</h2>
+        </span>
+    </header>
+      <!-- CUSTOM CARDS -->
+    <div class="custom-card-container">
+    {{#each section.sections}}
+    <div class="custom-contact-card-wrapper">
+         <section class="custom-card small-card">
+                {{#is id 1500002293041}} {{!-- User Support Email --}}
```

Comment:
I feel if you just contain all inside one `#is` block it would be easier to add or remove articles. There is more typing, especially the `h3` part, but easier to maintain in the future.
```
 {{#is id 1500002293041}}
  <a href="mailto:cetsupport@configura" class="custom-card-link">
    <div class="custom-card-logo-container">
      <i class="material-icons black-icon">textsms</i>
    </div>
    <h3 class="custom-card-title">{{name}}</h3>
  </a>
{{else}}
...
{{/is}}
```